src/utils/svg.js

```diff
@@ -0,0 +1,199 @@
+import * as THREE from 'three'
+import {SVGLoader} from 'three/examples/jsm/loaders/SVGLoader'
+import {assertDefined} from './assert'
+import debug from './debug'
+
+
+const svgLoadingManager = new THREE.LoadingManager()
+const svgLoader = new SVGLoader(svgLoadingManager)
+const fileLoadingManager = new THREE.LoadingManager()
+const fileLoader = new THREE.FileLoader(fileLoadingManager)
+
+
+export const getSVGGroup = async ({
```

Comment:
Please use function with jsdoc for top-level funcs

Reply:
done